fake-news-detection/src/fake_news_detection/pipelines/data_science/pipeline.py

```
@@ -32,5 +32,4 @@ def create_pipeline(**kwargs) -> Pipeline:
             inputs=["train_dataset", "val_dataset", "params:model_options"],
             outputs=["predictions", "evaluation_report"],
             name="train_and_evaluate_node",
-        ),
-    ])
+        ),
```

Ruff (0.11.9)

SyntaxError: unexpected EOF while parsing

---

No newline at end of file

(W292)